src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1039,6 +1039,12 @@ def _configure_security_policy_storage_params(arg_ctx):
                    help='Generate and assign an Azure Active Directory Identity for this server'
                    'for use with key management services like Azure KeyVault.')
 
+        c.argument('enable_public_network',
```

Comment:
1. Please add help.py example showing how to specify false.
2. Check with other RP's that their argument definition is the same, e.g. help text and `-e` shortcut
3. Mark the param as preview?